docker/postgres/postgres.conf

```diff
@@ -0,0 +1,25 @@
+listen_addresses = '*'
+
+max_connections = 100
+
+# should be enough for the performance server that has 8GB RAM and 700MB database
+# should be also fine for development
+shared_buffers = 1GB
+
+dynamic_shared_memory_type = posix
+
+# seeking random page is fast on SSD that is in the performance server
+random_page_cost = 1.0
+
+log_timezone = 'UTC'
+
+datestyle = 'iso, mdy'
+
+timezone = 'UTC'
+
+lc_messages = 'en_US.utf8'
+lc_monetary = 'en_US.utf8'
+lc_numeric = 'en_US.utf8'
+lc_time = 'en_US.utf8'
+
+default_text_search_config = 'pg_catalog.english'
```

Comment:
If the file in the monorepo and in the project-base is identical, I would suggest having it only in the project-base - just as we do with the php-fpm directory with its Dockerfile, php.ini and such...